README.md

```diff
@@ -20,7 +20,7 @@ For more details see [license.txt](https://github.com/LowPowerLab/RaspberryPi-Ga
 http://lowpowerlab.com/gateway
 
 ###Quick reference:
-- Copy index.php, graphHelper.js and images, sounds subfolders in your /var/www/default/ folder.
```

Comment:
I see now, but is there a particular reason for renaming to html?
I would prefer keeping .php to avoid any confusion.